HeavyFlavorAnalysis/Onia2MuMu/src/Onia2MuMuPAT.cc

```diff
@@ -134,8 +134,15 @@ Onia2MuMuPAT::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 	TransientVertex myVertex = vtxFitter.vertex(t_tks);
 
 	CachingVertex<5> VtxForInvMass = vtxFitter.vertex( t_tks );
-	Measurement1D MassWErr = massCalculator.invariantMass( VtxForInvMass, muMasses );
-	
+
+        Measurement1D MassWErr(jpsi.M(),-9999.);
```

Comment:
instead of try-catch - please just check the field value directly
@alberto-sanchez